Packs/ClarotyXDome/IncidentFields/incidentfield-Claroty_xDome_Device_Last_Seen.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_clarotyxdomedevicelastseen",
+    "version": -1,
+    "modified": "2024-04-18T10:35:36.815308343Z",
+    "name": "Claroty xDome Device Last Seen",
+    "ownerOnly": false,
+    "description": "The date and time a device's NIC was last seen",
+    "cliName": "clarotyxdomedevicelastseen",
+    "type": "multiSelect",
```

Comment:
is this should be multiselect?

Reply:
I chose to make it an `Array (Dynamic list)` because it looked like the most suitable option.
![WhatsApp Image 2024-05-16 at 12 55 25](https://github.com/demisto/content/assets/126682705/ca5b32d3-2c63-4231-854f-16d46b3e1cff)

Although you're right to think that it is a single-value field (it mostly is), sometimes devices will have multiple NICs and therefor this fields will contain multiple values (Last Seen for each NIC).
Do you think I should've defined something differently?

Reply:
This depends on your values from the fetched JSON of the alert. If you are mapping multiple values use the multi-select option then :) just wanted to make sure that this is intentional.